src/behaviors/loading/demo/loading.module.js

```diff
@@ -0,0 +1,16 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('semantic.ui.angular.loading',[
+            'semantic.ui.behaviors.loading'
+        ]).run(function($rootScope) {
+
+          $rootScope.opts = {
```

Comment:
Wait, sorry my bad, thought this was part of the actual lirbary and not the demo. It's fine. But does anything actually use this?